edenia/eden-spend-explorer
# feat(webapp): sort funtionality for tables

### Sort funtionality for tables

### What does this PR do?

- Resolve #72 

### Steps to test

1. run the project and open localhost:3000 in your browser
2. Click on income in the sidebar
3. scroll down to the table and see the new changes

#### CheckList
